internal/cache/avi_ctrl_clients.go

```diff
@@ -55,7 +55,7 @@ func SharedAVIClients() *utils.AviRestClientPool {
 				connectionStatus = utils.AVIAPI_CONNECTED
 				if err != nil {
 					connectionStatus = utils.AVIAPI_DISCONNECTED
-					utils.AviLog.Error("AVI controller initilization failed")
+					utils.AviLog.Fatal("AVI controller initilization failed")
```

Comment:
if you do a fatal here, won't it crash the pod?

Reply:
instead relying on the liveness probe is a better option?

Reply:
thanks, made the changes. It should shutdown the apiserver and the liveness probe will fail.